### PR TITLE
add arm64 architecture packages, and fix Architecture field of deb

### PR DESCRIPTION
**Pull request by lufia**
I added recipes to make the packages for arm64 architecture, and fixed Architecture of deb to be correct.

see mackerelio/mkr#310

### CODE REVIEW

**Comment by lufia** [comment]
### v2-centos

```console
[root@2fa2e589bccd x86_64]# rpm -ivh mackerel-agent-plugins-0.62.0-1.el7.centos.x86_64.rpm 
Preparing...                          ################################# [100%]
Updating / installing...
   1:mackerel-agent-plugins-0.62.0-1.e################################# [100%]
[root@2fa2e589bccd x86_64]# yum info mackerel-agent-plugins
Loaded plugins: fastestmirror, ovl
Determining fastest mirrors
 * base: ftp.nara.wide.ad.jp
 * extras: ftp.nara.wide.ad.jp
 * updates: ftp.nara.wide.ad.jp
base                                                                   | 3.6 kB  00:00:00     
extras                                                                 | 2.9 kB  00:00:00     
updates                                                                | 2.9 kB  00:00:00     
(1/4): base/7/x86_64/group_gz                                          | 153 kB  00:00:00     
(2/4): extras/7/x86_64/primary_db                                      | 206 kB  00:00:00     
(3/4): base/7/x86_64/primary_db                                        | 6.1 MB  00:00:03     
(4/4): updates/7/x86_64/primary_db                                     | 4.5 MB  00:00:03     
Installed Packages
Name        : mackerel-agent-plugins
Arch        : x86_64
Version     : 0.62.0
Release     : 1.el7.centos
Size        : 25 M
Repo        : installed
Summary     : Monitoring program metric plugins for Mackerel
URL         : https://mackerel.io/
License     : ASL 2.0
Description : This package provides metric plugins for Mackerel.
```

### v1-x86_64

```console
[root@c887099a438f x86_64]# rpm -ivh mackerel-agent-plugins-0.62.0-1.x86_64.rpm 
Preparing...                          ################################# [100%]
Updating / installing...
   1:mackerel-agent-plugins-0.62.0-1  ################################# [100%]
[root@c887099a438f x86_64]# yum info mackerel-agent-plugins
Loaded plugins: fastestmirror, ovl
Loading mirror speeds from cached hostfile
 * base: ftp-srv2.kddilabs.jp
 * extras: ftp-srv2.kddilabs.jp
 * updates: ftp-srv2.kddilabs.jp
Installed Packages
Name        : mackerel-agent-plugins
Arch        : x86_64
Version     : 0.62.0
Release     : 1
Size        : 287 M
Repo        : installed
Summary     : Monitoring program plugins for Mackerel
URL         : https://mackerel.io/
License     : Apache-2
Description : This package provides plugins for Mackerel.
```

### v1-noarch

```console
[root@c887099a438f noarch]# rpm -ivh mackerel-agent-plugins-0.62.0-1.noarch.rpm 
Preparing...                          ################################# [100%]
Updating / installing...
   1:mackerel-agent-plugins-0.62.0-1  ################################# [100%]
[root@c887099a438f noarch]# yum info mackerel-agent-plugins
Loaded plugins: fastestmirror, ovl
Determining fastest mirrors
 * base: ftp-srv2.kddilabs.jp
 * extras: ftp-srv2.kddilabs.jp
 * updates: ftp-srv2.kddilabs.jp
base                                                                   | 3.6 kB  00:00:00     
extras                                                                 | 2.9 kB  00:00:00     
updates                                                                | 2.9 kB  00:00:00     
(1/4): base/7/x86_64/group_gz                                          | 153 kB  00:00:00     
(2/4): extras/7/x86_64/primary_db                                      | 206 kB  00:00:00     
(3/4): updates/7/x86_64/primary_db                                     | 4.5 MB  00:00:02     
(4/4): base/7/x86_64/primary_db                                        | 6.1 MB  00:00:03     
Installed Packages
Name        : mackerel-agent-plugins
Arch        : noarch
Version     : 0.62.0
Release     : 1
Size        : 239 M
Repo        : installed
Summary     : Monitoring program plugins for Mackerel
URL         : https://mackerel.io/
License     : Apache-2
Description : This package provides plugins for Mackerel.
```

**Comment by lufia** [comment]
## deb

### v1
```console
root@cd934f5ecba6:/data/agent-plugins# dpkg -i mackerel-agent-plugins_0.62.0-1_all.deb 
Selecting previously unselected package mackerel-agent-plugins.
(Reading database ... 6503 files and directories currently installed.)
Preparing to unpack mackerel-agent-plugins_0.62.0-1_all.deb ...
Unpacking mackerel-agent-plugins (0.62.0-1) ...
Setting up mackerel-agent-plugins (0.62.0-1) ...
root@cd934f5ecba6:/data/agent-plugins# dpkg --info mackerel-agent-plugins_0.62.0-1_all.deb 
 new debian package, version 2.0.
 size 38579748 bytes: control archive=1968 bytes.
     207 bytes,     8 lines      control              
    3947 bytes,    58 lines      md5sums              
 Package: mackerel-agent-plugins
 Version: 0.62.0-1
 Architecture: all
 Maintainer: Shinji Tanaka <stanaka@hatena.ne.jp>
 Installed-Size: 244972
 Section: misc
 Priority: optional
 Description: plugins for Mackerel
```

### v2-amd64
```console
root@cd934f5ecba6:/data/agent-plugins# dpkg -i mackerel-agent-plugins_0.62.0-1.v2_amd64.deb 
Selecting previously unselected package mackerel-agent-plugins.
(Reading database ... 6503 files and directories currently installed.)
Preparing to unpack mackerel-agent-plugins_0.62.0-1.v2_amd64.deb ...
Unpacking mackerel-agent-plugins (0.62.0-1.v2) ...
Setting up mackerel-agent-plugins (0.62.0-1.v2) ...
root@cd934f5ecba6:/data/agent-plugins# dpkg --info mackerel-agent-plugins_0.62.0-1.v2_amd64.deb 
 new debian package, version 2.0.
 size 5734192 bytes: control archive=576 bytes.
     326 bytes,    10 lines      control              
     230 bytes,     3 lines      md5sums              
 Package: mackerel-agent-plugins
 Version: 0.62.0-1.v2
 Architecture: amd64
 Maintainer: Mackerel Developers <mackerel-developers@hatena.ne.jp>
 Installed-Size: 26026
 Depends: libc6 (>= 2.3.2)
 Section: misc
 Priority: optional
 Description: metric plugins for mackerel-agent
  metric plugins for https://mackerel.io (Monitoring SaaS)
```

File: Makefile [hunk 1]
```diff
@@ -22,11 +22,16 @@ build:
 	  $(MAKE) $(BINDIR)/$$i; \
 	done
 
-build/mackerel-plugin:
+build/mackerel-plugin: $(patsubst %,depends_on,$(GOOS)$(GOARCH))
 	mkdir -p build
+	rm -f build/mackerel-plugin
```
Comment (shibayu36) [inline Makefile:27]: Do you need this code?  I think we need either `$(patsubst %,depends_on,$(GOOS)$(GOARCH))` or `rm -f build/mackerel-plugin`

**Review by shibayu36 — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (lufia) [reply Makefile:27]: No, I will drop this line.

**Pull request merged**